convert_to_tf_keras.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+list_conversions = [('import keras.', 'import tensorflow.keras.'),

```

Comment:
Instead of doing text replace on the code base, cant we do this in a more programmatic/pythonic way? Just like how we handle multiple backends in keras?

Here is what I propose:

1 - We have a config.json in user/.keras-contrib which mentions whether to use keras or tf.keras
2 - We have a single place in the codebase which handles this duality . Lets call it `keras_lib.py`:

```python
keras_to_use = read('config.json')

if keras_to_use == "keras":
     module = "keras"
elif keras_to_use == "tf.keras":
     module = "tensorflow.keras"

globals().update(importlib.import_module(module).__dict__)
```

3 - Import keras_lib instead of keras everywhere else in the codebase
```python
from .keras_lib import Dense

class MyCustomDense(Dense):
    pass
```

4 - Profit!!

Reply:
This is fine by me as long as it doesn't confuse too much the contributors.
Let's say I'm a contributor and I want to simply do:
```python
from keras.layers import Conv2D
import keras.backend as K
from keras.callbacks import TensorBoard
```

what would it look like with your proposition? It's not clear to me.

Reply:
I might add that I'm not familiar with those python imports tricks so it's a bit hard for me to follow.

Reply:
```python
from keras_contrib.keras_lib.layers import Conv2D
from keras_contrib.keras_lib import backend as K
from keras_contrib.keras_lib.callbacks import TensorBoard
```

(You can also relative import keras_lib since its in the same package)

Reply:
I'm a bit worried that contributors would read the code and think that they have to access the layers of keras contrib by doing `from keras_contrib.keras_lib.layers`.

Also when a contributor will read 
`from keras_contrib.keras_lib import backend as K` he/she will wonder what's going on.

If a normal reader enter a file in a codebase and can't even understand the first lines written, I believe it's going to slow down contributions. Readers will have to be aware of this hack to understand imports.

Reply:
At first, I wanted a fork of the codebase for tf.keras, but I believe this fork can be generated automatically, also, this fork can be tested in travis, to ensure that we don't catch bugs too late.

Reply:
Agree. It does look confusing. Lets do it your way,

Reply:
Cool, I'll continue working on it. I hope to finish by this weekend.